pkg/util/http.go

```diff
@@ -40,8 +40,10 @@ func CreateHTTPClient(timeout time.Duration, unsafeSsl bool) *http.Client {
 	httpClient := &http.Client{
 		Timeout: timeout,
 		Transport: &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: unsafeSsl},
-			Proxy:           http.ProxyFromEnvironment,
+			TLSClientConfig:   &tls.Config{InsecureSkipVerify: unsafeSsl},
+			Proxy:             http.ProxyFromEnvironment,
+			DisableKeepAlives: true,
+			IdleConnTimeout:   100 * time.Second,
```

Comment:
I think we should have the option to configure this. I mean, now always we will want to disable the keep alive, working with some scaler we could be interested on it.
IMO we should add an option to configure this, keeping the old behavior as default